shared/reducers/wallets.js

```diff
@@ -263,6 +263,10 @@ export default function(state: Types.State = initialState, action: WalletsGen.Ac
       return state.merge({
         acceptingDisclaimerDelay: true,
       })
+    case WalletsGen.loadedMobileOnlyMode:
+      return state.withMutations(stateMutable => {
```

Comment:
can just be
`return state.update('mobileOnlyMap', map => map.set(action.payload.accountID, action.payload.enabled))`
or
`return state.updateIn(['mobileOnlyMap', action.payload.accountID], action.payload.enabled)`